service/authorization/v2/authorization.proto

```diff
@@ -0,0 +1,161 @@
+// 3. one resource
+message GetDecisionRequest {
+  // entity must be provided
+  EntityReference entity = 1 [(buf.validate.field).required = true];
+  // name on action is required
+  policy.Action action = 2 [(buf.validate.field).required = true];
+  Resource resource = 3 [(buf.validate.field).required = true];
+
+  option (buf.validate.message).cel = {
+    id: "get_decision_request.action_name_required"
+    message: "action.name must be provided"
+    expression: "has(this.action.name)"
+  };
+}
+message GetDecisionResponse {
+  // decision on the resource
+  ResourceDecision decision = 1;
+  // optional list of obligations represented in URI format
+  // repeated string obligations = 2;
+}
+
+// Can the entity (PE or NPE) access?
+// 1. one entity chain
+// 2. one action
+// 3. multiple resources
+// Note: this is a more performant bulk request for multiple resource decisions
+message GetDecisionMultiResourceRequest {
+  // entity must be provided
+  EntityReference entity = 1 [(buf.validate.field).required = true];
+  // name on action is required
+  policy.Action action = 2 [(buf.validate.field).required = true];
+  repeated Resource resources = 3 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).repeated = {min_items: 1}
+  ];
+
+  option (buf.validate.message).cel = {
+    id: "get_decision_multi_request.action_name_required"
+    message: "action.name must be provided"
+    expression: "has(this.action.name)"
+  };
+}
+message GetDecisionMultiResourceResponse {
+  // convenience flag indicating global resource decisions result (permit/deny)
+  google.protobuf.BoolValue all_permitted = 1;
+  // individual resource decisions
+  repeated ResourceDecision resource_decisions = 2;
+}
+
+// Can the entities (PE or NPE) access?
+// 1. Multiplexing of a Decision<entityChain, Action, repeated Resource> request
+// This is a more performant bulk request for complex decisioning (multiple entity chains or actions on
+// multiple resource decisions)
+message GetDecisionBulkRequest {
+  repeated GetDecisionMultiResourceRequest decision_requests = 1;
+}
+message GetDecisionBulkResponse {
+  repeated GetDecisionMultiResourceResponse decision_responses = 1;
+}
+
+// What can the entity (PE or NPE, represented by the entity chain) do, broken down as permitted actions on attribute value FQNs?
+//
+// Note: the v1 API parameter 'scope' has been dropped, and it is recommended to use
+// GetDecision if the resource is known
+message GetEntitlementsRequest {
+  // entity must be provided
+  EntityReference entity = 1 [(buf.validate.field).required = true];
+  // optional parameter to return all entitled values for attribute definitions with hierarchy rules, propagating
+  // down the hierarchical values instead of returning solely the value that is directly entitled
+  optional bool with_comprehensive_hierarchy = 2;
+}
+message GetEntitlementsResponse {
+  repeated EntityEntitlements entitlements = 1;
+}
+
+service AuthorizationService {
```

Comment:
Removed `ByToken` RPCs since we're already going to be switching on the per entity type in each RPC to accommodate registered resources, so might as well just capture token in the same logic.